Cargo.toml

```diff
@@ -16,3 +16,4 @@ default = ["serde", "stable_deref_trait", "std"]
 memoffset = "0.6"
 serde = { version = "1.0", default-features = false, optional = true }
 stable_deref_trait = { version = "1.1.1", default-features = false, optional = true }
+unsize = "1.1"
```

Comment:
Can this be an optional dependency? You can then make CI use `--all-features`

Reply:
CI seems to already be running all-features.